.github/workflows/build.yml

```diff
@@ -17,10 +17,8 @@ jobs:
         node-version: 16.x
     - name: Install dependencies
       run: npm install
-    - name: Build Sass
-      run: npx sass --load-path=node_modules --quiet-deps application.scss docs/application.css
     - name: Build Eleventy
-      run: npm run eleventy -- --output=docs
+      run: npm run build
```

Comment:
@paulrobertlloyd I think this will still need to output to the `docs` subfolder, as GitHub Pages only seems to let you deploy from that folder or root?